source/appModules/powerpnt.py

```diff
@@ -412,18 +412,18 @@ def _get_firstChild(self):
 
 	def handleSelectionChange(self):
 		"""Pushes focus to the newly selected object."""
-		if getattr(self,"_isHandlingSelectionChange",False):
```

Comment:
What exactly is the change here and over the next several lines? I'm struggling to read the diff.

Reply:
My mistake, whenever I edit this file my IDE normalizes the line endings. This file uses a mix of line endings. I'm going to open a separate PR to normalize this file, as this isn't the first time I have run into this.

Reply:
`git diff --ignore-cr-at-eol source/appModules/powerpnt.py`